src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier ColorSupplier = new ColorSupplier();
```

Comment:
The naming convention for variables is to start with a lowercase letter. Consider renaming 'ColorSupplier' to 'colorSupplier' to follow Java naming conventions.